experimental/packages/api-logs/src/types/LogRecord.ts

```diff
@@ -73,7 +73,7 @@ export interface LogRecord {
   /**
    * A value containing the body of the log record.
    */
-  body?: string;
+  body?: LogAttributeValue;
```

Comment:
While this is technically correct, we want to avoid the use of the term `attribute` for `body` as it has been confusing during the diccussions. It would be good to define another type to represent map + primitive types or even rename LogAttributeValue.

Reply:
I have added an alias for the type `EventDataValue`